### PR TITLE
Updates to building and exporting

### PR DESCRIPTION
**Pull request by mattgrill**
@gabesullice 

This changes how things work very slightly.
- Building is now handled with `Webpack`.
  - Use `npm run build` to create a packaged copy of `Drupal-api` for use in browser environments.
  - When loading the built version of `Drupal-api`, `window.Drupalapi` is automatically available. 
- Requiring `drupal-api` now works correctly as the documentation describes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 node_modules
-.watch
```
Comment (gabesullice) [inline .gitignore:2]: I would prefer to keep the Makefile and my little watch script in for running my tests automatically.

File: Makefile [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-BUILD_DIRECTORY=build
-DIST_DIRECTORY=dist
-TESTS_DIRECTORY=tests
-
-UGLIFY_COMMAND=$(shell pwd)/node_modules/uglify-js/bin/uglifyjs
-TEST_COMMAND=$(shell pwd)/node_modules/mocha/bin/mocha $(TESTS_DIRECTORY)
-
-UGLIFY_OPTIONS=--compress
-
-build: $(DIST_DIRECTORY)/entityQuery.min.js $(DIST_DIRECTORY)/dapi.min.js $(DIST_DIRECTORY)/app.min.js
```
Comment (gabesullice) [inline Makefile:11]: This can just be updated to `npm run build` if necessary

File: package.json [hunk 3]
```diff
@@ -1,26 +1,29 @@
...
+    {
+      "name": "Matthew Grill",
+      "email": "matt.grill@acquia.com"
+    }
+  ],
   "license": "ISC",
   "devDependencies": {
-    "browserify": "^13.0.1",
     "chai": "^3.5.0",
+    "expose-loader": "^0.7.1",
```
Comment (gabesullice) [inline package.json:25]: Do you know what this is for?

Comment (mattgrill) [inline package.json:25]: `expose-loader` is used in the build process for Webpack. It enables Drupal-api to be [exposed automatically](https://github.com/gabesullice/dapi.js/pull/1/files#diff-0b8371040a48919248df224b81605655R4) when using it in a browser environment.

**Comment by gabesullice** [comment]
Thanks!

**Pull request merged**